addon-test-support/ember-cli-qunit.js

```diff
@@ -0,0 +1,65 @@
+import QUnit from 'qunit';
+import AbstractTestLoader, {
+  addModuleExcludeMatcher,
+  addModuleIncludeMatcher
+} from 'ember-cli-test-loader/test-support/index';
+
+
+addModuleExcludeMatcher(function(moduleName) {
+  return QUnit.urlParams.nolint &&
+    moduleName.match(/\.(jshint|lint-test)$/);
+});
+
+addModuleIncludeMatcher(function(moduleName) {
+  return moduleName.match(/\.jshint$/);
+});
+
+let moduleLoadFailures = [];
+
+QUnit.done(function() {
+  if (moduleLoadFailures.length) {
+    throw new Error('\n' + moduleLoadFailures.join('\n'));
+  }
+});
+
+export class TestLoader extends AbstractTestLoader {
```

Comment:
Definitely is, just unsure if it breaks any assumptions in the ember-exam code (been awhile since I looked at it haha)